src/omnisharp/OmnisharpManager.ts

```diff
@@ -32,24 +31,31 @@ export class OmnisharpManager {
                 throw new Error('Invalid path specified');
```

Comment:
Is this really a condition where we need to throw an exception? We know exactly what went wrong...